docs/archived/locked-inc-ftp/figures/firmware/source/tasks/watchdog_task.c

```
@@ -340,29 +339,26 @@ esp_err_t watchdog_task_stop(void)
-bool watchdog_task_is_running(void)
-{
+bool watchdog_task_is_running(void) {
   return s_g_context_initialized && s_task_context.task_running &&
          s_task_context.task_handle != NULL;
 }
 
-bool watchdog_task_get_stats(uint32_t*                health_checks_performed,
-                             uint32_t*                recovery_actions_taken,
-                             watchdog_health_level_t* last_health_level)
-{
+bool watchdog_task_get_stats(uint32_t *health_checks_performed,
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'watchdog_task_get_stats' of similar type ('int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)